packages/amplication-client/src/Resource/create-resource/wizard-pages/LabelDescriptionSelector.tsx

```diff
@@ -65,15 +65,7 @@ export const LabelDescriptionSelector: React.FC<
         </CreateServiceCircleBadge>
       )}
 
-      {image && (
-        <CreateServiceCircleBadge>
-          <img
-            className={classNames(`${className}__logo`, imageSize)}
-            src={image}
-            alt={""}
-          />
-        </CreateServiceCircleBadge>
-      )}
+      {image && <CreateServiceCircleBadge>{image}</CreateServiceCircleBadge>}
```

Comment:
if you want to preserve the old behaviour you need to keep it in case image is string

Reply:
We don't need it anymore, I fixed all places using this @barshimi